zib/StructureDefinitions/zib-Patient.xml

```diff
@@ -0,0 +1,330 @@
+        <description value="GeslachtCodelijst" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.0.1.1--20240415000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-patient-v4.3-2024EN" />
+        <map value="NL-CM:0.1.9" />
+        <comment value="Gender" />
+      </mapping>
+    </element>
+    <element id="Patient.birthDate">
+      <path value="Patient.birthDate" />
+      <short value="DateOfBirth" />
+      <definition value="Patient’s date of birth. The date of birth is mandatory for a patient. A vague date (such as only the year) is permitted." />
+      <alias value="Geboortedatum" />
+      <mapping>
+        <identity value="zib-patient-v4.3-2024EN" />
+        <map value="NL-CM:0.1.10" />
+        <comment value="DateOfBirth" />
+      </mapping>
+    </element>
+    <element id="Patient.deceased[x]">
+      <path value="Patient.deceased[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Patient.deceased[x]:deceasedBoolean">
+      <path value="Patient.deceased[x]" />
+      <sliceName value="deceasedBoolean" />
+      <short value="DeathIndicator" />
+      <definition value="An indication stating whether the patient has died." />
+      <alias value="OverlijdensIndicator" />
+      <type>
+        <code value="boolean" />
+      </type>
+      <mapping>
+        <identity value="zib-patient-v4.3-2024EN" />
+        <map value="NL-CM:0.1.32" />
+        <comment value="DeathIndicator" />
+      </mapping>
+    </element>
+    <element id="Patient.deceased[x]:deceasedDateTime">
+      <path value="Patient.deceased[x]" />
+      <sliceName value="deceasedDateTime" />
+      <short value="DateOfDeath" />
+      <definition value="The date on which the patient died. A ‘vague’ date, such as only the year, is permitted." />
+      <comment value="The zib recognizes both the DeathIndicator (boolean) and DateOfDeath (datetime) concepts, whereas in FHIR only one of these concepts may be present. Therefore, if deceasedDateTime is present and has a value, its value can be mapped to DateOfDeath and DeathIndicator is assumed to be true." />
+      <alias value="DatumOverlijden" />
+      <alias value="OverlijdensIndicator" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-patient-v4.3-2024EN" />
+        <map value="NL-CM:0.1.33" />
+        <comment value="DateOfDeath" />
+      </mapping>
+      <mapping>
+        <identity value="zib-patient-v4.3-2024EN" />
+        <map value="NL-CM:0.1.32" />
+        <comment value="DeathIndicator" />
+      </mapping>
+    </element>
+    <element id="Patient.address">
+      <path value="Patient.address" />
+      <definition value="Patient's address information." />
+      <type>
```

Comment:
Hier zou denk ik eigenlijk wel de alias gezet kunnen worden van NL-CM:0.1.4 --> zoals we ook doen voor ContactInformation bijv. Dan kan dat stukje ook uit de known-issues.yml

Reply:
Zodra zib-Patient en zib-AddressInformation in dezelfde branch zitten, worden de `short` en `alias` weggegooid uit de differential, en dan heb je waarschijnlijk wel weer een known issue nodig.